pkgs/by-name/fe/fertilizer/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "fertilizer";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "moleculekayak";
+    repo = "fertilizer";
+    tag = "v${version}";
+    hash = "sha256-sDoAjEiKxHf+HtFLZr6RwuXN+rl0ZQnFUoQ09QiE6Xc=";
+  };
+
+  build-system = with python3.pkgs; [ setuptools ];
+
+  dependencies = with python3.pkgs; [
+    bencoder
+    colorama
+    flask
+    requests
+  ];
+
+  pythonImportsCheck = [ "fertilizer" ];
+
```

Comment:
Tests? (Either set `doCheck = false;` or add say `pytestCheckHook` to `nativeCheckInputs`)

Reply:
Ah yeah, forgot to add the hook, thanks :-)